### PR TITLE
Allow custom APPID

### PR DESCRIPTION
**Pull request by Wil32**
Allow custom APPID

### CODE REVIEW

**Comment by mbround18** [comment]
What is the point of a custom app I'd? This tool is designed for Valheim and not another app.

**Comment by Wil32** [comment]
It would allow to quickly switch to Beta(1223920) and back for testing. I understand this would only help a minority.

**Comment by mbround18** [comment]
I see, I did not know they had a beta branch for the dedicated server.

File: src/scripts/start_valheim.sh [hunk 1]
```diff
@@ -32,7 +32,7 @@ Password: (REDACTED)
 Auto Update: ${AUTO_UPDATE}
 "
 
-export SteamAppId=892970
+export SteamAppId=${APPID}
```
Comment (mbround18) [inline src/scripts/start_valheim.sh:35]:
```sh
export SteamAppId=${APPID:-892970}
```

**Review by mbround18 — CHANGES_REQUESTED** [review]
Please set a default for this env variable

**Comment by Wil32** [comment]
> I see, I did not know they had a beta branch for the dedicated server.
Actually there is no Beta Server available at this moment. Only Beta client. Those AppID need to match in order to see each other.

**Review by mbround18 — APPROVED** [review]

**Pull request merged**